projects/demo-playwright/tests/kit/tiles/tiles.pw.spec.ts

```diff
@@ -0,0 +1,19 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe('Tiles', () => {
+    test('correctly loose box shadow after dragged', async ({page}) => {
+        await tuiGoto(page, '/components/tiles#vertical');
+
+        const example = new TuiDocumentationPagePO(page).getExample('#vertical');
+        const first = example.locator('tui-tile').first();
+        const last = example.locator('tui-tile').last();
+
+        await expect(example).toHaveScreenshot('01-tiles-drag-and-drop.png');
```

Comment:
![01-tiles-drag-and-drop-chromium-darwin](https://github.com/user-attachments/assets/bd6cd620-42c3-4035-b442-01a5cc202d8c)